scripts/configure.js

```diff
@@ -508,7 +512,12 @@ const getConfig = (() => {
           },
         },
         windows: {
-          files: {},
+          files: {
+            ".gitignorewindows":
```

Comment:
Why was this renamed? I don't think Git will pick it up if you do.
```suggestion
            ".gitignore":
```

Reply:
Oh, yea. This is all new to me so I just got confused and I feel stupid now.
Obviously, we have to have the name `.gitignore` in order to git to pick it up but it doesn't seem to copy contents from `example/windows/.gitignore`
What do you want me to do?

Reply:
I made the changes locally and I can see the two new files just fine. How are you testing it? Can you push your changes so we can take a closer look?